Tests/RxCocoaTests/NSSegmentedControl+RxTests.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  NSSegmentedControl+RxTests.swift
+//  Tests
+//
+//  Created by Mykola Voronin on 10/3/17.
+//  Copyright © 2017 Krunoslav Zaher. All rights reserved.
+//
+
+import RxSwift
+import RxCocoa
+import AppKit
+import XCTest
+
+final class NSSegmentedControlTests: RxTest {
+    
```

Comment:
can you add more tests please?

Reply:
@freak4pc I've added additional tests for getting and setting selected segment